Assets/Project/Scripts/UnityChanControlScript.cs

```diff
@@ -4,12 +4,21 @@
 
 namespace UnityChan
 {
+    public class Ability
+    {
+        public string AnimParamName { get; set; }
+        public KeyCode Key { get; set; }
+        public Action FuncToCall { get; set; }
+        public bool IsEnabled { get; set; }
+    }
+
     [RequireComponent(typeof(Animator))]
     [RequireComponent(typeof(CapsuleCollider))]
     [RequireComponent(typeof(Rigidbody))]
     public class UnityChanControlScript : MonoBehaviour
     {
         #region Fields
+        public bool IsDead { get; set; }
 
         //        
         public float lookSmoother = 3.0f;
```

Comment:
I think some of these public variables should be private or private with SerializeField tag

Reply:
yes I think most of this can actually just be deleted, its remnants of the original japanese code